project/Game.py

```diff
@@ -170,6 +174,24 @@ def modify_landmark(self, oldclue, clue=None, question=None, answer=None):
         except KeyError:
             return False
 
+    def edit_landmark_order(self, oldindex, newindex):
+        if not self.started:
```

Comment:
This should be a fail fast condition and also needs to check the ended status. So it should read something like:
`if self.started or self.ended: return Errors.CanOnlyDoThisWhenTheGameIsNew`
GAME_NOT_STARTED doesn't make sense in this context because we want the game in a not started state.